broker-daemon/broker-rpc/broker-rpc-server.js

```diff
@@ -26,6 +26,16 @@ const BROKER_PROTO_PATH = './broker-daemon/proto/broker.proto'
  */
 const IS_PRODUCTION = (process.env.NODE_ENV === 'production')
 
+const HTTP_PORT = 27591
```

Comment:
Lets add `const` comments here and default this value to `27592`. Its was a little confusing to have this mapped to a different port in `docker-compose` when it doesnt need to be (we only map different ports when there will be a conflict with other containers).